src/properties/property-accessors.html

```diff
@@ -27,6 +29,19 @@
     proto = Object.getPrototypeOf(proto);
   }
 
+  /**
+   * Used to save the value of a property that will be overridden with
```

Comment:
Used to save => Saves (for consistency with other methods).

"Will be overridden" is vague because it's not clear whether this is something that Polymer's doing, or something the user's doing.  ... Looks like if called from `_createPropertyAccessor` it's called immediately before Polymer overrides the accessor. (In which case, perhaps, "before Polymer overrides it with an accessor.")

But this is also a non-underscored method, which implies that people might call it directly under some circumstance...?

Also a little clearer if it sticks to present tense: will be => are/is 

Also it struck me as slightly inconsistent that property name was specified second here, but the other methods that operate on properties specify the property name first. (But this doesn't have "property" in the method name, either, so maybe that's irrelevant.)